packages/core/src/tracing/index.js

```diff
@@ -135,7 +133,6 @@ exports.sdk = sdk;
 exports.spanBuffer = spanBuffer;
 exports.supportedVersion = supportedVersion;
 exports.util = tracingUtil;
-exports.isLatestEsmSupportedVersion = isLatestEsmSupportedVersion;
```

Comment:
This was absolutely unnecessary. The ESM util can be accessed via `core.util.esm` now.